### DISCUSSION TITLE
Works on modded Nintendo Swtch?

### DISCUSSION THREAD

**Discussion by gusklein** [cwmonkey/nms-expeditions#17] [Q&A]
Any way to adapt the json file to use on a modded Nintendo Switch? Thanks!

**Comment by cwmonkey** [comment]
@gusklein I have no idea, but if you have access to the file system you can hunt around for the SEASON_DATA_CACHE.JSON, replace it, go offline and try it. You might want to check with Nintendo Switch modding communities to see if they have some idea of how to do that. Holler if you hear anything!

**Comment by cwmonkey** [comment]
For visibility, mentioning your instructions to get this working on Nintendo Switch here: https://github.com/leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/discussions/36#discussioncomment-11244832
